docs/ecosystem/ngsim.rst

```diff
@@ -0,0 +1,100 @@
+.. _ngsim:
```

Comment:
Document isn't included in any toctree. We may include it under the ecosystem toctree in index.rst .

Reply:
Done.